lib/world.rb

```diff
@@ -0,0 +1,40 @@
+class World
+
+  attr_reader :cells
+
+  def initialize(cells)
+    @cells = cells
+  end
+
+  def evolve
+    next_generation = []
+      @cells.each {|cell|
+        neighbours_count = count_live_neighbours(cell.position)
+        if neighbours_count == 2 || neighbours_count == 3
```

Comment:
Minor point to consider - you could put `neighbours_count == 3` into a method called `overcrowded?` and `neighbours_count == 2` into a method `underpopulated?` (or whatever they correspond to). 

This would give a little bit more context to the reader, rather than just using 'magic numbers'. The code would then read `if underpopulated? || overcrowded?` which might be nicer?